docs/operations/metrics.md

```diff
@@ -133,6 +133,7 @@ Most metric values reset each emission period, as specified in `druid.monitoring
 |`jetty/threadPool/min`|Number of minimum threads allocatable.|`druid.server.http.numThreads` plus a small fixed number of threads allocated for Jetty acceptors and selectors.|
 |`jetty/threadPool/max`|Number of maximum threads allocatable.|`druid.server.http.numThreads` plus a small fixed number of threads allocated for Jetty acceptors and selectors.|
 |`jetty/threadPool/queueSize`|Size of the worker queue.|Not much higher than `druid.server.http.queueSize`.|
+|`jetty/httpClient/threadpool/queueSize`|Size of the worker queue at jetty http client.|Less than or equal to 1024(default value).|
```

Comment:
How do you intend to use this metric?

Reply:
This is to monitor if we have a sufficient number of Connections per destination. Let's suppose the latency is high at the broker. It might be possible the request was waiting at the jetty client-side queue, causing the latency to go high.

Reply:
the jetty client module is only being used on coordinator and router though. And it's unusual that you are hitting a limit 1024.

Reply:
what service did you see this log line in 
```
 `jetty/httpClient/threadpool/queueSize` Size of the worker queue at jetty http client. Less than or equal to 1024(default value).
```

Reply:
I do have this from the router, but I remember seeing it on the broker(when it communicates with data nodes) let me see if I can again validate that. But i do have one example from router:
```
Max requests queued per destination 1024 exceeded for HttpDestination[$(broker)]@4b9ee947,queue=1024,pool=DuplexConnectionPool@7698916c[c=0/100/100,a=100,i=0,q=1024] 
```

Reply:
>the jetty client module is only being used on coordinator and router though.
But the broker also communicates with data nodes, so what does the broker use for communicating with data nodes?

Reply:
Actually, I am more interested in the broker-side client queue. Can you help me with which client we use there?

Reply:
https://github.com/apache/druid/blob/43d790fdb771ac05b66938746a99a8bc5bf7cc74/server/src/main/java/org/apache/druid/guice/http/HttpClientModule.java#L46

Reply:
@abhishekagarwal87, we are not setting the thread pool executor in the HttpClientModule. In the jetty server module, we also expose queue size from the thread pool; so is there any other way we can emit queue size here in HttpClientModule?